oem/firstvoices/android/build_keyboards.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/sh
+
+# Build the keyboards referred to in ../keyboards.csv and copy them + metadata 
+# to the FirstVoices/Keyboards folder
+
+# keyboards.csv has columns Shortname,ID,Name,Region,OldVersion
+
+# Set sensible script defaults:
+# set -e: Terminate script if a command returns an error
+set -e
+# set -u: Terminate script if an unset variable is used
+set -u
+# set -x: Debugging use, print each statement
+# set -x
+
+# This build script assumes that the https://github.com/keymanapp/keyboards repo is in
+# the same parent folder as this repo, with the default name 'keyboards'
+KEYBOARDS_ROOT=../../../../keyboards
+
+function die {
+  echo "FATAL: $1"
+  exit 99
+}
+
+function display_usage {
+  echo "Usage: build_keyboards.sh [-copy-keyboards] [-clean-keyboards] [-debug] [-h|-?]"
+  echo "Builds all keyboards used by the app and copies them into the"
+  echo "target path."
+  echo "  -copy-keyboards: Only copy the keyboards; don't rebuild them"
+  echo "  -clean-keyboards: Clean the keyboards from this repo"
+  echo "  -debug: Build debug versions of the keyboards"
+  exit 1
+}
+
+DO_CLEAN=true
+DO_COPY=true
+DO_BUILD=true
+
+while [[ $# -gt 0 ]] ; do
```

Comment:
I tried building this in Ubuntu 

1. I had to make the 3 .sh files executable `sudo chmod +x *.sh`
2. Is this line bash syntax?
```
while [[ $# -gt 0 ]] ; do
```

Reply:
and I had to make `gradlew` executable

Reply:
👍  all done.

Point 2 above? I've updated the relevant scripts to use `#!/bin/bash`. Were you having trouble with it in other shells?